### PR TITLE
More and better logs for geoserver_init

### PR DESCRIPTION
**Pull request by chrismayer**
Adds more logs for `geoserver_init` service in order to be able to do better debugging (in case of failures).

### CODE REVIEW

File: geoserver_init/010_init-gs.js [hunk 1]
```diff
@@ -96,13 +99,16 @@ async function createWorkspaces() {
...
     stationWorkspace, stationDataStore, pgHost, pgPort, pgUser, pgPassword,
     pgSchema, pgDb
   );
 
-  if (success) {
-    console.info('Successfully created PostGIS store');
+  if (created) {
+    console.info('Successfully created PostGIS store', stationDataStore);
+  } else {
+    console.error('Failed creating PostGIS store (maybe already existing ',
```
Comment (JakobMiksch) [inline geoserver_init/010_init-gs.js:110]: closing bracket is missing

```suggestion
    console.error('Failed creating PostGIS store (maybe already existing) ',
```

File: geoserver_init/010_init-gs.js [hunk 2]
```diff
@@ -69,7 +69,7 @@ async function initGeoserver() {
   if (proxyBaseUrlChanged) {
     console.info(`Set proxy base url to "${proxyBaseUrl}"`);
   } else {
-    console.info('Setting proxy base url failed.');
+    console.error('Failed setting proxy base url', proxyBaseUrl);
```
Comment (JakobMiksch) [inline geoserver_init/010_init-gs.js:72]: A whitespace is missing

```suggestion
    console.error('Failed setting proxy base url: ', proxyBaseUrl);
```

File: geoserver_init/010_init-gs.js [hunk 3]
```diff
@@ -86,6 +86,9 @@ async function createWorkspaces() {
     const wsCreated = await grc.workspaces.create(ws);
     if (wsCreated) {
       console.info('Successfully created workspace', wsCreated);
+    } else {
+      console.error('Failed creating workspace (maybe already existing ',
```
Comment (JakobMiksch) [inline geoserver_init/010_init-gs.js:90]: Add missing bracket

```suggestion
      console.error('Failed creating workspace (maybe already existing)  ',
```

**Review by JakobMiksch — APPROVED** [review]
@chrismayer looks fine. Some formatting/typos.
Once the project becomes bigger, we could consider using a logging library.

**Review by jdk24 — APPROVED** [review]
GeoServer init's [Log](https://github.com/meggsimum/sauber-sdi-docker/files/6676671/log_init.txt) looks good. Please merge at will!

**Comment by chrismayer** [comment]
Thanks for the reviews guys! Will merge now.

**Pull request merged**